roles/image_builder/README.md

```diff
@@ -0,0 +1,38 @@
+Role Name
```

Comment:
none of this content looks relevant to the role does, probably could use an update .... check out the role READMEs we did for the infra.osbuild collection for inspiration

Reply:
Sure thing, it seems like this is from the auto gen from the role command.

I'm on it